### DISCUSSION TITLE
Sharing resources between extension

### DISCUSSION THREAD

**Discussion by FredLL-Avaiga** [microsoft/vscode-discussions#272] [Extension Development QnA] [answered]
Hi,

I writing an extension that has a dependency on another extension (here even-better-toml)
That extension can take contribution from other extension via package.json / contributes/... (here tomlValidation)
it expects an URL so file:// would be fine
How can I specify a path to a resource file that is part of my extension ?
ie How can I use extensionUri in package.json ...
Tx 
Fred

**Comment by isidorn** [comment, marked as answer]
I do not think you can use the extensionUri in the package.json dynamically if that is your question.
